cmd/generate/action/template/cmd/flag.go

```diff
@@ -1,4 +1,4 @@
-package template
+package cmd
```

Comment:
The templates got moved due to the new structure we needed for the `pkg/action` templates.